cosmo_tester/framework/test_hosts.py

```diff
@@ -1369,3 +1390,21 @@ def _update_instance(self, server_index, node_instance):
             server_id,
             server_index,
         )
+
+    def _disable_ipv4(self, node_instances):
```

Comment:
This should be on the host itself in the finalise, then you don't need to separately wait_for_ssh, because you can put it on after that, and just make it auto-execute this if the test VM is marked as ipv6.

Reply:
Do you mean this part of code https://github.com/cloudify-cosmo/cloudify-system-tests/blob/f8fe18688e683375208adc48d9c452eb4ef4f39b/cosmo_tester/framework/test_hosts.py#L244-L245?  I am not sure we know IP addresses of other two VMs at this point.

Reply:
Ah, I see the problem then, yes. Fair enough. Can you add a comment noting that that is done in the test_hosts because the IPs of all hosts need to be known before applying the settings?